src/components/structures/UserSettings.js

```diff
@@ -82,7 +82,6 @@ const SIMPLE_SETTINGS = [
     { id: "VideoView.flipVideoHorizontally" },
     { id: "TagPanel.disableTagPanel" },
     { id: "enableWidgetScreenshots" },
-    { id: "RoomSubList.showEmpty" },
```

Comment:
this seems to have snuck in, but it and other related changes are their own commit so that's probably fine for now. In general, changes should be distinct PRs.

Reply:
Ah right, it happened to be something I noticed while working on this feature, but you're right it quite distinct. I'll make separate PRs in the future, thanks! 😄